http_test.go

```diff
@@ -14,11 +17,35 @@ func TestHttp(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = &httpSuite{}
+var _ = Suite(&httpSuite{})
 
 type httpSuite struct{}
 
-func (s httpSuite) TestServeHTTP(c *C) {
+type hasPrefix struct {
```

Comment:
This type name is a huge misnomer 🙂.

```suggestion
type contains struct {
```